plugins/python-build/scripts/add_miniconda.py

```diff
@@ -213,6 +234,13 @@ def make_script(specs: List[MinicondaSpec]):
 
 
 def get_existing_minicondas():
+    """
+    Get known miniconda versions
+    -----------------------
+
+    This function returns a generator of :class:`MinicondaVersion` objects for all files in the
+    output directory that start with ``miniconda`` and are not named ``latest``. The returned values are sorted by version number.
+    """
```

Comment:
Style is different from other docstrings. Header is inaccurate and probably superfluois, the explanation should probably be in user terms and not reciting the code; what's "output directory"?

Reply:
Fixed style, and improved wording. See latest.